Resources/Prototypes/_Impstation/Objectives/traitor.yml

```diff
@@ -0,0 +1,99 @@
+#SVS objectives
+
+- type: entity
+  parent: [BaseTraitorObjective, BaseKillObjective]
+  id: KillRandomTraitorSvSObjective # Used for rounds when basically every valid person will be a traitor.
+  description: Do it however you like, just make sure they don't make it to centcom.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: true
+  - type: TargetObjective
+    title: objective-condition-kill-person-title
+  - type: PickRandomTraitor
+  - type: ObjectiveBlacklistRequirement
+    blacklist:
+      component: null
+
+- type: entity
+  parent: [BaseTraitorSvSSocialObjective, BaseKeepAliveObjective]
+  id: RandomTraitorAliveSvSObjective # Used for rounds when basically every valid person will be a traitor.
+  description: Identify yourself at your own risk. We just need them alive.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: true
+  - type: TargetObjective
+    title: objective-condition-other-traitor-alive-title
+  - type: RandomTraitorAlive
+  - type: ObjectiveBlacklistRequirement
+    blacklist:
+      component: null
+
+#steal
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: FinFinMeatStealObjective
+  components:
+  - type: ObjectiveLimit
+    limit: 4
+  - type: StealCondition
+    stealGroup: FoodMeatFinfin
```

Comment:
i'm not sure if can roll without verifyMapExistence set to false, and even with it turned off: this is really just an objective to rng into a fin fin spawn, otherwise you have to earn 100k for cargo. really not ideal